app/presenters/task_list/collection.rb

```diff
@@ -1,5 +1,5 @@
 module TaskList
-  class Collection < ::BasePresenter
+  class Collection < SimpleDelegator
```

Comment:
I've reverted this to not subclass from `BasePresenter` and use just the pure `SimpleDelegator` like before, as this presenter is kind of a one-off and don't require some of the functionality pushed to the `BasePresenter` (like the `present` method, etc. and maybe more in the future).